logcheck/pkg/logcheck.go

```diff
@@ -36,6 +36,7 @@ const (
 	parametersCheck  = "parameters"
 	contextualCheck  = "contextual"
 	withHelpersCheck = "with-helpers"
+	verboseZeroCheck = "verbose-zero"
```

Comment:
One last thing: should this be "verbosity" instead of "verbose"?

I think `V` stands for "verbosity" of the log message, but I might be wrong (not a native speaker).

A log message is "verbose", but it has "verbosity zero".